### ISSUE TITLE
VS2022: msbuild can not be located, only Build Tools are installed

### ISSUE THREAD

**Issue by nils-a** [cake-build/cake#3671]
As discussed in https://github.com/cake-build/cake/discussions/3476,
VS2022 BuildTools should be searched in `Program Files (x86)`, too.

**Comment by DrewRhoades** [comment]
It sure would have been nice to see this fix in a Cake 1.3.1 release.  My Cake 1.3.0 build doesn't recognize the VS2022 Build Tools on my build server, and in order to fix it, I have to rework all my build script to be compatible with Cake.Tools 2.0.0, which appears to be non-trivial (I'm currently using the full framework runner).  So I'm basically stuck without investing significant effort.

**Comment by DrewRhoades** [comment]
In case someone comes behind me looking for a work-around, I basically used a method as suggested here: https://github.com/cake-build/cake/discussions/3476

I got the MSBuild path using VSWhere:

```
#tool nuget:?package=vswhere&version=2.8.4

    public FilePath LatestMSBuildPath { get; private set; }

    LatestMSBuildPath = GetLatestMSBuildPath();

    private FilePath GetLatestMSBuildPath()
    {
        var latestMSBuildPath = context.VSWhereLatest(new VSWhereLatestSettings { Products = "*", Requires = "Microsoft.Component.MSBuild"});
        return new FilePath($"{latestMSBuildPath}\\MSBuild\\Current\\Bin\\MSBuild.exe");
    }
```

And then when compiling for VS2022, I override the ToolPath in my MSBuildSettings:

```
    private Func<MSBuildSettings, DirectoryPath, MSBuildSettings> GetMSBuildSettings()
    {
        return (msbuildSettings, path) =>
        {
            msbuildSettings
                .WithTarget("Build")

			...

			if (this.VSVersion == MSBuildToolVersion.VS2022)
			{
				context.Information($"MSBuildPath: {this.LatestMSBuildPath}");
				msbuildSettings.ToolPath = this.LatestMSBuildPath;
			}
			
            return msbuildSettings;
        };
    }
```
